### PR TITLE
add call activity linking page to web modeler doc

### PR DESCRIPTION
**Pull request by nithinssabu**
Closes https://github.com/camunda/web-modeler/issues/1566

### CODE REVIEW

**Comment by CatalinaMoisuc** [comment]
As agreed with @jfriedenstab let's get this internally reviewed before handing it over to DevRel.

File: sidebars.js
Comment (christian-konrad) [inline sidebars.js:115]: We should not hide it under "Additional features". This could mean everything that we haven't found the right place for. It should be more like "Advanced modeling"

File: docs/components/modeler/web-modeler/additional-features/call-activity-linking.md
Comment (christian-konrad) [inline docs/components/modeler/web-modeler/additional-features/call-activity-linking.md:13]: Can we use better examples in the screenshot (Something real-world like "Invoke payment" or similar, especially having both callee and caller here is confusing, would just expect the callee to appear-I know self-calling is possible but it is still not the best case to show)

Comment (christian-konrad) [reply docs/components/modeler/web-modeler/additional-features/call-activity-linking.md:13]: Best would be to use names of processes that re-appear in the docs, so we have one consistent "storyline" throughout the docs

File: docs/components/modeler/web-modeler/additional-features/call-activity-linking.md
Comment (christian-konrad) [inline docs/components/modeler/web-modeler/additional-features/call-activity-linking.md:17]: Can we also add a real-world name here in the image

File: docs/components/modeler/web-modeler/additional-features/call-activity-linking.md
Comment (christian-konrad) [inline docs/components/modeler/web-modeler/additional-features/call-activity-linking.md:23]: Can we use a more friendly (fake) process id here in the screenshot?

File: docs/components/modeler/web-modeler/additional-features/call-activity-linking.md
Comment (christian-konrad) [inline docs/components/modeler/web-modeler/additional-features/call-activity-linking.md:26]: Thanks for this notice!

File: docs/components/modeler/web-modeler/additional-features/call-activity-linking.md
Comment (christian-konrad) [inline docs/components/modeler/web-modeler/additional-features/call-activity-linking.md:21]: Add some reasoning: "... in the properties panel, e.g. when you call a process from another project."

File: docs/components/modeler/web-modeler/additional-features/call-activity-linking.md
Comment (christian-konrad) [inline docs/components/modeler/web-modeler/additional-features/call-activity-linking.md:7]: Please explain why the user should do that. What are the benefits of call activities? Why do I need to link the processes? Is this adding convenience (i.e. I do not need to enter process IDs)?
You may just write a short sentence why call activities are helpful (... to reduce complexity and support reusability, you can call another process as a building block using call activities... if the child process model is part of the project of your parent model, you can link them using following approaches...) and then reference the call activity doc as you did (https://docs.camunda.io/docs/components/modeler/bpmn/call-activities/)

Comment (christian-konrad) [reply docs/components/modeler/web-modeler/additional-features/call-activity-linking.md:7]: (I also think the sentence is semantically wrong: you don't link the call activity to another process, but you link a child process via a call activity to another process.)

File: docs/components/modeler/web-modeler/additional-features/call-activity-linking.md
Comment (christian-konrad) [inline docs/components/modeler/web-modeler/additional-features/call-activity-linking.md:11]: ...create the link by _pasting_ the process ID of the diagram to...

Comment (christian-konrad) [reply docs/components/modeler/web-modeler/additional-features/call-activity-linking.md:11]: Howevery, this is a very transparent explanation of how this is achieved, great!

File: docs/components/modeler/web-modeler/additional-features/call-activity-linking.md
Comment (christian-konrad) [inline docs/components/modeler/web-modeler/additional-features/call-activity-linking.md:15]: This does not cover the (edge) case of multiple files with same process ids found. Do we want to cover edge cases in the docs in general?
Also, it should be "...which shows _the name of the diagram_ that the call activity _links to_."

**Review by christian-konrad — CHANGES_REQUESTED** [review]
I like the transparency of this docs entry. However, we should address some issues before releasing it into the docs

File: docs/components/modeler/web-modeler/additional-features/call-activity-linking.md
Comment (CatalinaMoisuc) [reply docs/components/modeler/web-modeler/additional-features/call-activity-linking.md:11]: Would be great if we could align this with other parts of the documentation that describe a step by step guide on how to perform an action:

https://docs.camunda.io/docs/components/modeler/web-modeler/collaboration/#inviting-users-to-projects

https://docs.camunda.io/docs/components/modeler/desktop-modeler/model-your-first-diagram/

https://docs.camunda.io/docs/components/modeler/web-modeler/launch-cloud-modeler/

https://docs.camunda.io/docs/guides/model-your-first-process/#design-and-deploy-a-process

File: sidebars.js
Comment (CatalinaMoisuc) [reply sidebars.js:115]: What about the "Collaboration", "Milestones", "Token Simulation" pages? I think the section is getting too big.

Comment (christian-konrad) [reply sidebars.js:115]: Good catch @CatalinaMoisuc, we could put "Token Simulation" under "Validate your models", and "Milestones" could belong to somewhere else, or both under "Advanded modeling". We shouldn't do it in the scope of this issue, but a general overhaul could help us to keeping a nice-to-read documentation. I would recommend that there should not be more than 7 parent sections

Comment (CatalinaMoisuc) [reply sidebars.js:115]: @christian-konrad What about the "Collaboration", "Milestones", "Token Simulation" pages? I think the section is getting too big.

Comment (CatalinaMoisuc) [reply sidebars.js:115]: Makes sense. Once we have these changes out, we can open a new PR to reorganize the pages. cc @nithinssabu

File: docs/components/modeler/web-modeler/additional-features/call-activity-linking.md
Comment (nithinssabu) [reply docs/components/modeler/web-modeler/additional-features/call-activity-linking.md:21]: I'm not sure if we should add such a reasoning. This is just another way of linking and can be used also to link a process from the samem project.

File: docs/components/modeler/web-modeler/additional-features/call-activity-linking.md
Comment (nithinssabu) [reply docs/components/modeler/web-modeler/additional-features/call-activity-linking.md:7]: In that case, does it make sense to call this feature Call activity linking?

Also, should this section be used to describe the benefits of call activities? Shouldn't it be in the call activity doc (like it is now) and shouldn't we use this section to explain how to create the link?

File: docs/components/modeler/web-modeler/additional-features/call-activity-linking.md
Comment (nithinssabu) [reply docs/components/modeler/web-modeler/additional-features/call-activity-linking.md:15]: I tried to keep the doc simple and included the bare minimum necessary for a user to understand how to use this feature. For the edge cases, we have included contextual help in the application in the form of `info` messages that appear when the user encounters them.

File: docs/components/modeler/web-modeler/additional-features/call-activity-linking.md
Comment (nithinssabu) [reply docs/components/modeler/web-modeler/additional-features/call-activity-linking.md:11]: If we think in terms of the `copy` and `paste` operations, the word _pasting_ makes sense.

But when explaining in English, isn't to correct to say _copying_?

File: docs/components/modeler/web-modeler/additional-features/call-activity-linking.md
Comment (nithinssabu) [reply docs/components/modeler/web-modeler/additional-features/call-activity-linking.md]: I changed the text to:
_You can use one of the following approaches to link the process to be called by a call activity_.

Is this better?

File: sidebars.js
Comment (nithinssabu) [reply sidebars.js]: https://github.com/camunda/web-modeler/issues/1777

File: docs/components/modeler/web-modeler/advanced-modeling/call-activity-linking.md
Comment (CatalinaMoisuc) [inline docs/components/modeler/web-modeler/advanced-modeling/call-activity-linking.md:7]: @nithinssabu which page should this link display? Maybe I am missing something, but I can't find it.

Comment (CatalinaMoisuc) [reply docs/components/modeler/web-modeler/advanced-modeling/call-activity-linking.md:7]: Nevermind, found it: https://docs.camunda.io/docs/components/modeler/bpmn/call-activities/.

File: docs/components/modeler/web-modeler/additional-features/call-activity-linking.md
Comment (christian-konrad) [reply docs/components/modeler/web-modeler/additional-features/call-activity-linking.md]: I am note 100% sure, this is what my grammar check gave me

File: docs/components/modeler/web-modeler/advanced-modeling/call-activity-linking.md
Comment (christinaausley) [inline docs/components/modeler/web-modeler/advanced-modeling/call-activity-linking.md:12]: I would specify what button you are referring to here. For example, "Click the link button and choose any diagram..."

**Review by christinaausley — APPROVED** [review]
Adjusted UI references from code block to bolded and made the description a little more clear. One comment for clarification.

File: docs/components/modeler/web-modeler/additional-features/call-activity-linking.md
Comment (christinaausley) [reply docs/components/modeler/web-modeler/additional-features/call-activity-linking.md]: How is it written is correct. "Copied to" is essentially "brought over to/duplicated in".

File: docs/components/modeler/web-modeler/additional-features/call-activity-linking.md
Comment (christinaausley) [reply docs/components/modeler/web-modeler/additional-features/call-activity-linking.md]: I was also confused by this. Perhaps "Use one of the following approaches to link one process to another using a call activity."?

File: docs/components/modeler/web-modeler/advanced-modeling/call-activity-linking.md
Comment (nithinssabu) [reply docs/components/modeler/web-modeler/advanced-modeling/call-activity-linking.md:12]: The next button also mentions clicking on the `Link` button which is a different link button. I didn't want to say "Click the link button on 2 consecutive lines, which could make it confusing.

**Comment by nithinssabu** [comment]
@christian-konrad Since you requested changes, it needs your approval to be merged. Please approve if you are happy with the current state.

**Review by christian-konrad — APPROVED** [review]
Looking great and helpful for the reader!

**Pull request merged**
